app/views/brexit_checker/_stay_updated.html.erb

```diff
@@ -1,10 +1,10 @@
-<h2 class="govuk-heading-l stay-updated__heading"><%= t("brexit_checker.stay_updated.heading") %></h2>
+<h2 class="govuk-heading-m stay-updated__heading">
+  <%= t("brexit_checker.stay_updated.heading") %>
+</h2>
 
-<div class="govuk-!-margin-top-4">
-  <%= render "govuk_publishing_components/components/govspeak", {
-    content: t("brexit_checker.stay_updated.description").html_safe
-  } %>
-</div>
+<%= render "govuk_publishing_components/components/govspeak", {
```

Comment:
I'm not sure this needs to be govspeak - it could just be a paragraph?

Reply:
good catch, amended